### PR TITLE
Update hivesmodals.html

### PR DESCRIPTION
**Pull request by ElectronEsq**
lbry.science just directs to an unrelated, broken/incomplete website. Not sure if LBRY Science is even a thing or not, so didn't remove it completely.
LBRY Records website no longer exists.

### CODE REVIEW

File: _includes/hivesmodals.html [hunk 1]
```diff
@@ -142,10 +142,7 @@ <h3 class="modal-title" id="LBRY_ScienceModal">LBRY.Science</h3>
         </div>
         <div class="modal-body">
           <h4>Chat Room or Forum</h4>
-          <p>LBRY Discord !Science role</p> 
-
```
Comment (Coolguy3289) [inline _includes/hivesmodals.html:146]: To my knowledge, lbry.science and BOINC are one and the same. JRingo runs this hive and it is still active last I checked. Please revert this change and restore the site link.

File: _includes/hivesmodals.html [hunk 2]
```diff
@@ -207,9 +204,6 @@ <h3 class="modal-title" id="LBRY_recordsModal">LBRY.records</h3>
           <h4>Chat Room or Forum</h4>
           <p>LBRY Discord !Musician role</p> 
 
-          <h4>Website</h4>
-          <p><a href="https://bitrophone.com/">https://bitrophone.com/</a></p>  
-
```
Comment (Coolguy3289) [inline _includes/hivesmodals.html:212]: LGTM

**Review by Coolguy3289 — CHANGES_REQUESTED** [review]

File: _includes/hivesmodals.html (hunk 1 above)
Comment (ElectronEsq) [reply _includes/hivesmodals.html:146]: The link isn't anything to do with LBRY. It's just a Boinc site. Furthermore the site doesn't have an SSL and is using dummy text. It's best to remove this, then if the site becomes a fully working LBRY related site at a later date, add it back then. As it stands it has no purpose on the site and reflects badly on the brand.

image

image

Comment (Coolguy3289) [reply _includes/hivesmodals.html:146]: Just confirmed, that site is correct for lbry.science

**Comment by jring-o** [comment]
Go ahead and remove it. It's a sandbox for the time being.

**Comment by Coolguy3289** [comment]
Just resolve the conflicts and I'll get this merged

**Comment by Eniamza** [comment]
@Coolguy3289 Ready to merge I guess

**Comment by Coolguy3289** [comment]
No, because of the upstream merge. The !role command documentation has been added again, as well as the website that Jon approved removing.

**Comment by Eniamza** [comment]
Ah,  I probably reverted messed up something.
Checking...

**Comment by Eniamza** [comment]
Updated it manually

**Pull request closed without merging**